include/execution.hpp

```diff
@@ -2892,6 +3132,15 @@ namespace _P2300::execution {
         noexcept(nothrow_tag_invocable<ensure_started_t, _Sender>) {
         return tag_invoke(ensure_started_t{}, (_Sender&&) __sndr);
       }
+      template <sender _Sender>
+        requires (!__tag_invocable_with_completion_scheduler<ensure_started_t, set_value_t, _Sender>) &&
+          (!tag_invocable<ensure_started_t, _Sender>)
+      __sender<_Sender> operator()(_Sender&& __sndr) const {
```

Comment:
There's also a minor optimization opportunity with making `ensure_started` the identity function when the given sender is already an `ensure_started` sender.